Theme1/src/firsttheme/PalindromeCheck.java

```diff
@@ -0,0 +1,38 @@
+package firsttheme;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class PalindromeCheck {
+    private static final Pattern pattern = Pattern.compile("\\W");
+    private static Matcher matcher;
+    public static int countOfPalindromes(String strings) {
+        int retval = 0;
+        if (strings.equals("")) return retval;
```

Comment:
it is a good practice to use `if` with brackets in any case. Please avoid using short variant of `if` clause